src/hotspot/share/gc/shared/generationCounters.hpp

```diff
@@ -54,25 +48,14 @@ class GenerationCounters: public CHeapObj<mtGC> {
 
   char*              _name_space;
 
-  // This constructor is only meant for use with the PSGenerationCounters
-  // constructor. The need for such an constructor should be eliminated
-  // when VirtualSpace and PSVirtualSpace are unified.
-  GenerationCounters()
-             : _current_size(nullptr), _virtual_space(nullptr), _name_space(nullptr) {}
-
-  // This constructor is used for subclasses that do not have a space
-  // associated with them (e.g, in G1).
-  GenerationCounters(const char* name, int ordinal, int spaces,
-                     size_t min_capacity, size_t max_capacity,
-                     size_t curr_capacity);
-
  public:
   GenerationCounters(const char* name, int ordinal, int spaces,
-                     size_t min_capacity, size_t max_capacity, VirtualSpace* v);
+                   size_t min_capacity, size_t max_capacity,
+                   size_t curr_capacity);
```

Comment:
Some parameters or arguments formats, like this one, can be adjusted/aligned. It seems an IDE automatical alignment. So, it is not a big problem.